kdl_parser/test/test_inertia_rpy.cpp

```diff
@@ -37,14 +37,12 @@
 #include <iostream>
 #include <vector>
 
-#include <gtest/gtest.h>
-#include <kdl/jntarray.hpp>
-#include <kdl/chainidsolver_recursive_newton_euler.hpp>
-
-#include <ros/ros.h>
-#include <ros/console.h>
 #include "kdl_parser/kdl_parser.hpp"
 
+#include "gtest/gtest.h"
+#include "kdl/jntarray.hpp"
+#include "kdl/chainidsolver_recursive_newton_euler.hpp"
```

Comment:
this looks like a style change but given that I consider this more a weird behavior of our config of cpplint rather than an actual Google Styleguide violation I think it's fine to not upstream.
Nit: out of alphabetical order